src/actions/searchActions.js

```diff
@@ -50,7 +50,7 @@ export const getFilteredArticles = () => (dispatch, getState) => {
     }
   });
 
-  return axios.post(`https://a-haven-staging.herokuapp.com/api/v1/articles/search/filter?searchQuery=${searchQuery}&limit=2000`, object)
+  return axios.post(`http://a-haven-staging.herokuapp.com/api/v1/articles/search/filter?searchQuery=${searchQuery}&limit=2000`, object)
```

Comment:
Nice work Kafi. I observed you changed `https` to `http`. Probably an oversight. Please look into it.

Reply:
Thanks for the observation. I have made a correction.